OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -2305,7 +2305,7 @@ public void run() {
       if (shouldLogUserPrivacyConsentErrorMessageForMethodName(OSTaskController.SET_SUBSCRIPTION))
          return;
 
-      getCurrentSubscriptionState(appContext).setUserSubscriptionSetting(disable);
+      getCurrentSubscriptionState(appContext).setPushDisabled(!disable);
```

Comment:
with the properties names changes, is the 
`public static void disablePush(final boolean disable)` method readable? 

The comments say

   * You can call this method with {@code false} to opt users out of receiving all notifications through
    * OneSignal. You can pass {@code true} later to opt users back into notifications.
    * @param disable whether to subscribe the user to notifications or not

shouldn't true be for opt users out of receiving all notifications through and false otherwise?

Reply:
ah yes, the comment is backwards and needs to be updated.